docker/Dockerfile

```diff
@@ -159,6 +163,7 @@ RUN addgroup -g ${GID} ${USER} && adduser -u ${UID} -D -G ${USER} -g "" ${USER}
     if [ "$ENABLE_JNI" = "ON" ]; then apk add --no-cache openjdk8-jre-base; fi && \
     if [ "$ENABLE_PCAP" = "ON" ]; then apk add --no-cache libpcap; fi && \
     if [ "$ENABLE_USB_CAMERA" = "ON" ]; then apk add --no-cache libpng libusb; fi && \
+    if [ "$ENABLE_LUA_SCRIPTING" = "ON" ]; then apk add --no-cache lua; fi && \
     if [ "$ENABLE_OPENCV" = "ON" ] || [ "$ENABLE_BUSTACHE" = "ON" ]; then apk add --no-cache boost; fi && \
     if { [ "$ENABLE_PYTHON" = "ON" ] || [ "$ENABLE_SCRIPTING" = "ON" ]; } && [ -z "$DISABLE_PYTHON_SCRIPTING" ]; then apk add --no-cache python3-dev; fi
```

Comment:
Not strictly related to this PR, but python3-dev should not be needed here, only the python3 package. Also ENABLE_PYTHON check can be removed here as it turns out that it's unrelated to the python scripting extension. (It's an unused nanofi based extension.)

Reply:
good idea changed it in https://github.com/apache/nifi-minifi-cpp/pull/1244/commits/c222bf84612ca4ff4bd85c6cf5dc79b03cbc1fed